docs/community/KPI-options.md

```diff
@@ -0,0 +1,43 @@
+---
+title: KPI Options
+sidebar_label: KPI Options
+---
+## KPI Option Overview
+
+Key Performance Indicator (KPI) Options are way for your protocol to encourage your community to participate in its growth by creating aligned incentives, where your community benefits from achieving your performance targets.
+
+The aligned incentive means that your community's assistance in helping you reach your goals is rewarded and by working together all community members benefit by your protocol's growth.
+
+### What are KPI options? 
+
+Using UMA’s Expiring Multi Party (EMP) contract template, any project can create an incentive mechanism which offers rewards to their targeted community in exchange for the community’s support in helping them achieve their Key Performance Metrics. 
+
+At a high-level, KPI options are synthetic ERC20 tokens that can be redeemed, but only if the project hits it’s predetermined KPI targets before expiry. These targets are up to each project. TVL Options for DeFi protocols, Volume Options for exchange protocols or DAU Options for dapps
+
+Every community member airdropped these KPI options, becoming vested in helping the project succeed on its Key Metrics, as the token grows more valuable as the target approaches.  This provides powerful incentive alignment between the protocol and its community. 
+
+### Launch your own KPI option
+
+The first step is [create an UMIP](https://docs.umaproject.org/uma-tokenholders/umips) to add your token to our list of accepted collateral currencies (if it isn’t already). Then you'll need to define the Key Performance Metrics. This can be any metric that is important to you, and that you think the community can help you achieve.
+
+Once the necessary UMIPs have been passed, the project will launch the contract.  After, the collateral tokens will locked in the contract and mint the KPI Option tokens. 
```

Comment:
I'd suggest unifying the person that this is written in. Some of this uses third person (the project) while other parts use "you"/"your project". Personally, I like the latter better.

Personally I think this section could use just a little bit of elaboration on what the "contract deployment" process entails. Wdyt about this? 

```suggestion
Once the necessary UMIPs have been passed, your project will be able to deploy an expiring KPI options contract. This process is extremely simple, and requires no contract development work or experience! [Here](https://docs.umaproject.org/developers/emp-deployment) is a tutorial to walk you through the contract deployment process. After deployment, you will be able to lock your collateral in the contract and mint your KPI Option tokens. 
```